### PR TITLE
SDK-1985 fetching session config support example

### PR DESCRIPTION
**Pull request by dhivya-pattabhiraman**
Updated the IDV example to retrieve and display the session configuration in a new tab.

### CODE REVIEW

File: examples/idv/views/pages/index.ejs [hunk 1]
```diff
@@ -1,3 +1,7 @@
 <%- include('layout/header'); -%>
+<div style="font-size: 18px; margin-left: 300px">
+    <span>Session ID:</span> <span><%= sessionId %></span>
+</div>
+<a style="font-size: 18px; margin-left: 300px" href="/config" target="_blank">View session configuration</a>
```
Comment (laurent-yoti) [inline examples/idv/views/pages/index.ejs:5]: Is that on purpose that the href is not including the `sessionID` query params ?

**Review by abdalmajeed-yoti — APPROVED** [review]

File: examples/idv/views/pages/index.ejs (hunk 1 above)
Comment (dhivya-pattabhiraman) [reply examples/idv/views/pages/index.ejs:5]: Yeah, the sessionID was not needed in the href.

Comment (dhivya-pattabhiraman) [reply examples/idv/views/pages/index.ejs:5]: Actually added the query param.

**Review by laurent-yoti — APPROVED** [review]
Looks great, please merge.

**Pull request merged**